packages/wallet/src/KeyManagement/util/constants.ts

```diff
@@ -11,3 +11,8 @@ export enum Cip1852PathLevelIndexes {
   // address
   INDEX = 4
 }
+
+export enum VotingLabels {
```

Comment:
I'm not sure this belongs under `KeyManagement/util` - it's specific to constructing cip-compliant transaction, not key management in general. How about moving cip36-specific stuff under `src/cip36`?